bagua/torch_api/distributed.py

```diff
@@ -59,32 +59,63 @@ def __init__(
         else:
             self.parameters_to_ignore = []
 
-    def unwarp(self):
+    def unwrap(self):
+        r"""
+        Return the unwraped module.
+        """
+
         return self.module
 
     def forward(self, *inputs, **kwargs):
+        r"""
+        Execute the forward process and return the output.
+        """
+
         result = self.module(*inputs, **kwargs)
         return result
 
 
 class Reducer(object):
-    """
-    -- New --
-    bucket_initialized is False:
-        1. add_param
-        2. initialize_buckets -> register_models
-        3. mark_bucket_ready
-        4. mark_on_complete
-
-    bucket_initialized is True:
-        1. mark_tensor_ready
-        2. mark_on_complete
+    r"""In order to improve communication efficiency, the distributed algorithm divides the parameter into different blocks, 
```

Comment:
algorithm "chunks parameters into many buckets. A bucket is the minimum unit of communication between devices in bagua."